src/Microsoft.Framework.TestHost/Program.cs

```diff
@@ -131,10 +128,11 @@ public int Main(string[] args)
                     var testServices = TestServices.CreateTestServices(_services, project, channel);
                     await ProjectCommand.Execute(testServices, project, args.ToArray());
                 }
-                catch
+                catch (Exception ex)
                 {
-                    // For now we're not doing anything with these exceptions, we might want to report them
-                    // to VS.   
+                    Trace.TraceError(ex.ToString());
+                    channel.SendError(ex);
+                    return -2;
```

Comment:
Note that since this happens inside the `using(var channel...)` we still wait for ACK (with a timeout).